packages/viewer/src/store/modules/ui/actions/setCompareSliderActive.ts

```diff
@@ -0,0 +1,10 @@
+import type { UIStore } from '@/store/modules/ui/types/ui'
+import type { ActionDispatcher } from '@/store/types'
+
+export default function setCompareSliderActive(
+    this: UIStore,
+    isActive: boolean,
+    dispatcher: ActionDispatcher
+): void {
+    this.isCompareSliderActive = !!isActive
```

Comment:
with TS doing the type checking for us, I think you can get rid of the `!!` here